src/schema_change_handler.cpp

```diff
@@ -109,9 +109,8 @@ void SchemaChangeHandler::on_set(const ResponseMap& responses) {
       has_error = true;
     }
   }
-  if (has_error) return;
 
-  if (has_schema_agreement(responses)) {
+  if (!has_error && has_schema_agreement(responses)) {
```

Comment:
@aholmberg An error is printed for each of the error responses in https://github.com/datastax/cpp-driver/blob/381/src/error_response.cpp#L210-Lundefined